submissions/krylenger/memory-pair-game/app.js

```diff
@@ -0,0 +1,244 @@
+const orelSong = new Audio('music/orel.mp3');
+const gopgopSong = new Audio('music/gopgop.mp3');
+const tanciSong = new Audio('music/tanci.mp3');
+const cadilacSong = new Audio('music/cadilac.mp3');
+const dimSong = new Audio('music/dim.mp3');
+
+let playedSongs = [];
+
+const cards = [
```

Comment:
I'd make this data more consistent. Some of the objects doesn't have the `text` and the `song` property.
It would be better to add this property to the each of the objects. You can set `text=''` where you don't use it

Reply:
✅ done